end-user-surveys/analysis-guides/survey-analysis.md

```diff
@@ -0,0 +1,189 @@
+# OpenTelemetry Survey Data Analysis Tutorial
+***A step-by-step walkthrough for analyzing survey data across SIGs in the OTel community.*** 
+
+***Author: Ernest Owojori***: ***Page 3 of 3***
+
+## Data Preparations
+Survey data often contains missing values, duplicates, or inconsistent category names, especially when questions allow open text or multiple responses. Start by getting a clear overview of what you’re working with.
+
+**Cleaning 1: Frequency & Percentage Check**
+
+Before editing anything, use Pivot Tables to generate frequency and percentage tables for each variable.
+This helps you quickly spot:
+
+ - Unfinished responses and missing values (blank cells)
+ - Inconsistent categories (e.g., “AWS Fargate” vs “aws fargate”)
+ - Outliers or unexpected entries
+
+***Why this matters: This simple step often surfaces most data issues early and informs how you’ll clean each variable***.
+
+Note: 
+
+ - Using pivot tables to get frequencies and percentages should not be used for multiple response variables and open-ended variables.
+ - Some blank cells can be due to the conditional formatting of the form.
+
+**Steps to using pivot tables:** 
```

Comment:
Double check, how many empty lines do you need here. I guess you might need one, but likely not more. This happens multiple times in the file.